App/Sources/DesignSystem/Extensions/View/MainRankingView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+struct MainRankingView: View {
+    let profileName: String = "안강호"
+    let ownPoint: String = "1000 M"
+    
+    var body: some View {
+        VStack {
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(spacing: 15.44) {
+                    ForEach(1..<4) { i in
```

Comment:
```suggestion
                    ForEach(1..<4) { ranking in
```
랭킹순으로 리스트가 들어오니까 i 보다는 ranking으로 받는게 더 가독성이 좋을 것 같아요

Reply:
넵 바로 수정하겠습니다~